www/templates/control/main-control.html

```diff
@@ -8,6 +8,10 @@
     <!-- each list item is very similar:
       contains a text and a button icon or switch (for the most part) 
       moving to react -- a text component and an icon button component that live in a list item comoponent? -->
+
+    <!-- here is the html related to my (failing) attempts to convert more of the profile screen -->
+    <setting-row text-key = "control.view-privacy" icon-name = "eye"></setting-row>
```

Comment:
At a high level, when you get to refactoring the profile screen, I think it might be worthwhile creating a parent React Component for the profile screen as well. If they all follow a similar pattern, we could even define the i18n label and the function to call in a map and then create the elements by iterating over the map!